### DISCUSSION TITLE
How to get models schema (type_defs) in from of string?

### DISCUSSION THREAD

**Discussion by AlenSci** [fastapi/fastapi#8645] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code
I have the following model.
```
class Post(Base):
    __tablename__ = "post"

    id = Column(Integer, primary_key=True, index=True)
    title = Column(String)
    author = Column(String)
    content = Column(String)
    time_created = Column(DateTime(timezone=True), server_default=func.now())
```
with this secham

```
class PostSchema(SQLAlchemyObjectType):
    class Meta:
        model = Post
```

### Description

* **Problem ** : How to to get the graphql schema definition in form of string like this
```
"""
type Post{
id: ID
title: string!
author: String!
content: String!
time_created: Int!
}
"""
```
* **I tried** this, but there is not `type_defs` option.
* `graphene.Field(PostModel).type_defs`

### Operating System

Linux, Windows, macOS, Other

### Operating System Details

any

### FastAPI Version

any

### Python Version

3.9

**Comment by nikeshnaik** [comment]
@AlenSci is this what you are looking for: 

```
schema {
  query: Query
}

type Query {
  users: [User]
}

type User {
  id: ID!
  name: String
  lastName: String
}
```
Followed these examples from graphene-sqlalchemy package: https://github.com/graphql-python/graphene-sqlalchemy/#examples 

**printing schema gave the output.**

```
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

class UserModel(Base):
    __tablename__ = 'user'
    id = Column(Integer, primary_key=True)
    name = Column(String)
    last_name = Column(String)




import graphene
from graphene_sqlalchemy import SQLAlchemyObjectType

class User(SQLAlchemyObjectType):
    class Meta:
        model = UserModel
        # use `only_fields` to only expose specific fields ie "name"
        # only_fields = ("name",)
        # use `exclude_fields` to exclude specific fields ie "last_name"
        # exclude_fields = ("last_name",)

class Query(graphene.ObjectType):
    users = graphene.List(User)

    def resolve_users(self, info):
        query = User.get_query(info)  # SQLAlchemy query
        return query.all()

schema = graphene.Schema(query=Query)

## this gave the output
print(schema)
```
